src/cdk/schematics/ng-generate/drag-drop/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -1,15 +1,27 @@
 <div class="container">
   <h2>To do</h2>
 
-  <cdk-drop #todoList [data]="todo" [connectedTo]="doneList" class="list" (dropped)="drop($event)">
+  <div
+    cdkDrop
```

Comment:
Should this be indented by four spaces? It feels odd that the attributes align with the child element.

See: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
I'm not sure about that. Should we be matching the indentation to other components in the repo or the Google guidelines?

Reply:
I don't think it matters for the other components. It just would be good if the components that will be generated by the schematics, are consistent and readable.

Reply:
Changed.